### PR TITLE
Classification (GMMbayes)

### PR DESCRIPTION
**Pull request by cdesira**
Updated GMMbayes. Replaced GMM() with updated scikit-learn GaussianMixture(). Subsequently performed transformations on the output.

### CODE REVIEW

**Comment by cdesira** [comment]
I've attempted to update GMMbayes to be compatible with the updated version of scikit-learn. I'd like to continue to help update astroML however my code is failing the checks.

**Comment by bsipocz** [comment]
This change is incompatible with <0.18 scikit-learn versions, meaning the decision in #114 is a blocker for this PR.

File: astroML/density_estimation/tests/test_density.py [hunk 1]
```diff
@@ -46,4 +47,4 @@ def test_gaussian1d():
     dx = x[1] - x[0]
     integral = np.sum(y*dx)
 
-    assert np.round(integral, 1) == 1
+    assert integral + epsilon >= 1.  # test that we are close to 1
```
Comment (bsipocz) [inline astroML/density_estimation/tests/test_density.py]: this isn't the right way to test this, and in fact the integral was much larger than 1 for this test.

**Comment by bsipocz** [comment]
#125 supersedes this one (it includes the contributions from here).

**Comment by bsipocz** [comment]
Thank you @cdesira  and @Engineero!

**Pull request closed without merging**